docs/data-management/properties.md

```diff
@@ -6,6 +6,55 @@ sidebar_position: 2
 
 ## Entity properties
 
-### Assignment properties
+## Assignment properties
 
 ## Metric properties
+
+A class of properties that is sourced from Fact SQL.
+
+### Adding via Fact Definition
+
+![Configuring a Fact SQL](/img/properties/metric-property-fact-sql.png)
+
+Add a Metric property to any of your existing Facts or by creating a new one.
```

Comment:
It may not be 100% clear how to do this from the pic and this description. Maybe add an arrow in the pic or say something like "After running your Fact SQL, select "Add a Property" under the *Add Properties* section and select any eligible column."